### PR TITLE
add new music for nuke

### PR DESCRIPTION
**Pull request by pofitlo**
## About the PR

add track Running out by Bolgarich that will be randomly selected between countdown

**Media**

https://www.youtube.com/watch?v=SzEp2nv6oZ4

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added new nuke music

### CODE REVIEW

**Comment by QuietlyWhisper** [comment]
I've used this song before, you might want to make sure it isn't abruptly silenced when the nuke goes off, if you didn't

**Comment by pofitlo** [comment]
@QuietlyWhisper no, there is all right. i cut and speed up this song a little. it take 1.50 minutes

File: Resources/Audio/StationEvents/attribution.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 countdown.ogg is created by qwertyquerty and licensed under CC-BY-SA-3.0. It is taken from https://github.com/BeeStation/BeeStation-Hornet at commit 79b8cc23cfb347cf23ea70fc5f6f39afedf9cde7.
+
+RunningOut is created by Bolgarich and licensed under CC-BY It is taken from https://www.youtube.com/watch?v=SzEp2nv6oZ4
```
Comment (vulppine) [inline Resources/Audio/StationEvents/attribution.txt:3]: need to change this to say CC-BY-3.0 (see: the license under the youtube video) and provide a link to it - here's a link to it: https://creativecommons.org/licenses/by/3.0/legalcode

**Review by vulppine — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
changelog should say nuke music not just "music"

File: Resources/Audio/StationEvents/attribution.txt (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Audio/StationEvents/attribution.txt]: Should be in attributions.yml, .txt is deprecated.

**Pull request merged**